atom/common/platform_util_mac.mm

```diff
@@ -252,4 +248,91 @@ void Beep() {
   NSBeep();
 }
 
+#if defined MAS_BUILD
```

Comment:
Maybe it would be cleaner to have:

```cpp
void SetProcessTitleActivityMonitor(const std::string& name) {
#if defined MAS_BUILD
  // NB: Can't be implemented with public APIs
#else
  // ...
#endif
}
```

Feel free to disregard this.